### PR TITLE
[PXCT-288][PXCT-251] Nano Matter Official - Content for launch - Product page + Technical documentation

### PR DESCRIPTION
**Pull request by mcmchris**
## What This PR Changes
- Product page update (community preview notes removed)
- Simple and full pinout update
- Block diagram and power tree update
- Datasheet update

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/03.nano/boards/nano-matter/datasheet/assets/Nano-Matter_Top.png
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/datasheet/assets/Nano-Matter_Top.png:1]: good

File: content/hardware/03.nano/boards/nano-matter/datasheet/assets/Nano_Matter_Block_Diagram-v2.png
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/datasheet/assets/Nano_Matter_Block_Diagram-v2.png:1]: Clock signal and antenna, added, good

File: content/hardware/03.nano/boards/nano-matter/datasheet/assets/Nano_Matter_Connectors.png
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/datasheet/assets/Nano_Matter_Connectors.png:1]: Image updated to latest version, good

File: content/hardware/03.nano/boards/nano-matter/datasheet/assets/Nano_Matter_Outline.png
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/datasheet/assets/Nano_Matter_Outline.png:1]: idem, good

File: content/hardware/03.nano/boards/nano-matter/datasheet/assets/Nano_Matter_PeripheralsActuators.png
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/datasheet/assets/Nano_Matter_PeripheralsActuators.png:1]: good

File: content/hardware/03.nano/boards/nano-matter/datasheet/assets/Nano_Matter_Pinout.png
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/datasheet/assets/Nano_Matter_Pinout.png:1]: feedback provided by message, improve margins to have the board more centered + add a little warning explaining information about the bottom solder pads can be found on the full pinout

File: content/hardware/03.nano/boards/nano-matter/datasheet/assets/Nano_Matter_Power_Tree_1.png
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/datasheet/assets/Nano_Matter_Power_Tree_1.png:1]: pending lates version **with date updated**

File: content/hardware/03.nano/boards/nano-matter/datasheet/datasheet.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ type: maker
 author: José Bagur
```
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/datasheet/datasheet.md:5]:
```suggestion
author: José Bagur, Christopher Méndez
```

File: content/hardware/03.nano/boards/nano-matter/datasheet/datasheet.md [hunk 2]
```diff
@@ -86,14 +86,13 @@ The table below provides a comprehensive guideline for the optimal use of the Na
 <div style="text-align:center;">
 
 |          **Parameter**           |   **Symbol**    | **Min** | **Typ** | **Max** | **Unit** |
-|:--------------------------------:|:---------------:|:-------:|:-------:|:-------:|:--------:|
-|     USB Supply Input Voltage     | V<sub>USB</sub> |    -    |   5.0   |    -    |    V     |
-| Supply Input Voltage<sup>1</sup> | V<sub>IN</sub>  |    -    |   5.0   |   5.5   |    V     |
+| :------------------------------: | :-------------: | :-----: | :-----: | :-----: | :------: |
+| Input voltage from USB connector | V<sub>USB</sub> |   4.8   |   5.0   |   5.5   |    V     |
+|    Input voltage from VIN pad    | V<sub>IN</sub>  |    6    |   7.0   |   21    |    V     |
```
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/datasheet/datasheet.md:91]: great thank you

**Review by jcarolinares — CHANGES_REQUESTED** [review]
@mcmchris **Datasheet review done** , please ping me when that part is ready

File: content/hardware/03.nano/boards/nano-matter/features.md
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/features.md:1]: checked

File: content/hardware/03.nano/boards/nano-matter/image.svg
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/image.svg:1]: Image updated, checked

File: content/hardware/03.nano/boards/nano-matter/interactive/ABX00112-pinout.png
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/interactive/ABX00112-pinout.png:1]: waiting latest pinout @mcmchris

File: content/hardware/03.nano/boards/nano-matter/datasheet/assets/Nano_Matter_Power_Tree_1.png
Comment (mcmchris) [reply content/hardware/03.nano/boards/nano-matter/datasheet/assets/Nano_Matter_Power_Tree_1.png:1]: Done

File: content/hardware/03.nano/boards/nano-matter/product.md
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/product.md:1]: good, checked

File: content/hardware/03.nano/boards/nano-matter/datasheet/datasheet.md (hunk 1 above)
Comment (mcmchris) [reply content/hardware/03.nano/boards/nano-matter/datasheet/datasheet.md]: Done

File: content/hardware/03.nano/boards/nano-matter/tech-specs.yml
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/tech-specs.yml:1]: all correct, checked

**Comment by mcmchris** [comment]
@jcarolinares can go forward with this, everything ready.

**Comment by jcarolinares** [comment]
> @jcarolinares can go forward with this, everything ready.

Datasheet changes addressed, **Datasheet is ready for launch, updating jira task**

File: content/hardware/03.nano/boards/nano-matter/interactive/ABX00112-pinout.png
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/interactive/ABX00112-pinout.png:1]: now yes :)

**Comment by jcarolinares** [comment]
This pull request is **ready**, thefore datasheet, product page/technical documentation is ready

**Review by jcarolinares — APPROVED** [review]
approved to be merge in other branches for launch

**Comment by jcarolinares** [comment]
@mcmchris this is ready, I'm going to merge it in the official launch branch and pull request, also renamed to avoid confussion

**Pull request merged**